docs/getting-started/fullstack-example.md

```diff
@@ -0,0 +1,79 @@
+
+## Create a KintoHub Account
+
+If you have not created a KintoHub account yet, please to create one [here](https://app.kintohub.com).
+
+## Deploy a Postgres Server
+
+To store our data, we will need a Postgres database.
+KintoHub provides a dedicated Postgres server with 1 Gb of storage.
+
+1. Click the **Create Service** At the top right of your environment
+2. Click the **From Catalog** tab
+3. Click **PostgreSQL**
+4. Enter or generate your `Username`, `Password`, and `Root Password`.
+5. Click **Deploy** At the top right.
+
+This process will take about **1 Minute** to deploy.
+Once the **Status** has changed from `Deploying` to `Live Version`, click on the **Access** tab at the top center of the page.
+
+Copy the **Connection String (Admin)** and paste it somewhere for setting up Hasura.
+
+## Deploy a Hasura Service
+
+This tutorial has a premade Hasura repository with migration files available.
+You can deploy it with the following steps:
+
+1. Click on **Create Service** at the top right
+2. Click **Web App**
+3. Copy and paste the following into the **Repository URL** `https://github.com/kintohub/todo-hasura`
+4. Click **Connect**
+5. Type `Dockerfile` in the **Dockerfile Name** field
+6. Type `8080` in the **Port** field
+7. Click the **Environment Variables** Tab
+8. Add the **key** `HASURA_GRAPHQL_DATABASE_URL` and paste in the **Connection String (Admin)** from the previous step as the **value**
+9. Click **Deploy** at the top right
+
+This will take around **30 Seconds** to complete.
+Once complete, you can click the link to the right of **Public API access via**
+You will now see Hasura's dashboard and can copy the **GraphQL Endpoint URL**
+
+## Setup React Web App
+
+This tutorial has a premade React Todo List App that works with Hasura.
+You can deploy it with the following stpes:
+
+1. Click on **Create Service** at the top right
+2. Click **Web App**
+3. Copy and paste the following into the **Repository URL** `https://github.com/kintohub/todo-react`
+4. Click **Connect**
+5. Type `Dockerfile` in the **Dockerfile Name** field
+6. Type `8080` in the **Port** field
+5. Click on **Environment Variables** Tab
+6. Add the **key** `REACT_APP_SERVER_URL` and paste the **GraphQL Endpoint URL** from the previous step as the **value**
+7. Click **Deploy**
+
+This will take around **3 Minutes** to complete.
+Once complete, you can click the link to the right of **Public API access via**
+
+## Optimize Cost
+
+**Congratulations! You have finished the full stack tutorial.**
+
+If you are on a paid account and have already used your monthly free credits, the total cost of this environment can be optimized to a max cost of **$4.34** USD per month.
+To do this, update your services with the following:
+
+* Lower the [memory](../anatomy/anatomy-advanced.md#memory) of **todo-hasura** from `256` to `128` MB of memory
+* Lower the [memory](../anatomy/anatomy-advanced.md#memory) of **postgres** from `256` to `128` MB of memory
```

Comment:
are we sure about that? uz right now @nandiheath needs to fix the limits on the catalog

Reply:
@Disturbing what I mean is that we have an issue atm, where the limits in not being set for catalog. So u can for example configure mongodb to run on 32MB while in overview it's actually consuming 200. It's a fix needed on the FE.